distribution/lib/Standard/Base/0.0.0-dev/src/Data/Vector/Extensions.enso

```diff
@@ -0,0 +1,86 @@
+import project.Any.Any
+import project.Data.Array.Array
+import project.Data.Array_Proxy.Array_Proxy
+import project.Data.Vector.Vector
+import project.Error.Error
+import project.Errors.Illegal_Argument.Illegal_Argument
+import project.Runtime
+
+## GROUP Selections
+   Swaps the rows and columns of a matrix represented by a `Vector` of `Vector`s.
+
+   ! Error Conditions
+
+     - If the rowws (subvectors) do not all have the same length, an
+     `Illegal_Argument` error is raised.
+
+   > Example
+     Transpose a `Vector` of `Vectors`.`
+
+         "Hello, world!".reverse
+
+         matrix = [[0, 1, 2], [3, 4, 5], [6, 7, 8]]
+         # +---+---+---+
+         # | 0 | 1 | 2 |
+         # +---+---+---+
+         # | 3 | 4 | 5 |
+         # +---+---+---+
+         # | 6 | 7 | 8 |
+         # +---+---+---+
```

Comment:
I really like your examples being so visual.

I think it will be great UX effort if we will be able to render them nicely in the GUI + if we can agree on a common format and be able to 'test' them verifying if the output is the same as we say it is.

Reply:
I agree, I very much want to run the examples as tests.